packages/opentelemetry-plugin-http/src/utils.ts

```diff
@@ -195,7 +208,7 @@ export class Utils {
     return { origin, pathname, method, optionsParsed };
   }
 
-  static getIncomingOptions(options: ParsedRequestOptions) {
+  static getOutgoingOptions(options: ParsedRequestOptions) {
```

Comment:
I think this function is no longer used (expect by tests). as followup also `hasExpectHeader` is no longer used then.